Automattic/wp-calypso
# Entrepreneur trial plan page

## Proposed Changes

![image](https://github.com/Automattic/wp-calypso/assets/33497086/74873b1a-d03d-4a56-91e2-dbcac16cc052)
We are introducing the new Entrepreneur trial plan, which share some similarities with Woo Express trial, but requires customized experience on Plans page.

Unlike the Woo Express trial, we are not going to offer the 1U$ offer when upgrading the Entrepreneur trial.

## Testing Instructions

For testing this you will need an Entrepreneur trial account:
p1712672819187649-slack-C02TCEHP3HA

You will also need a Woo Express trial to check that we didn't break the original experience
https://woocommerce.com/start/

Access `http://calypso.localhost:3000/plans/{your-site-here}`
- Check that the trial banner does not refer to the 1U$ offer.
- Click on **Add payment method**.
- You should be redirected to checkout page targeting the Entrepreneur plan and the 1U$ offer should **not** be offered.
- Go back to the `http://calypso.localhost:3000/plans/{your-site-here}`.
- On the bottom of the page there should be the **Keep all the features for your site** section.
- Select different billing periods, this should update the price of the plan inside the card.
- Click on the second button **Add payment method** button, the one inside the card.
- The checkout page billing period should match the one you selected before.

Do a regression test with Woo Express trial account and check that:
- The 1U$ offer is presented on the banner and checkout page.
- Checkout page should target `Woo Express: Performance` plan instead of `Entrepreneur.`

## Pre-merge Checklist
